src/ff8/mod.cpp

```diff
@@ -599,7 +599,7 @@ TexturePacker::TextureTypes TextureBackground::drawToImage(
 			const int col = tileId % _colsCount, row = tileId / _colsCount;
 
 			drawImage(
-				imgData, imgWidth / imgScale, imgScale,
+				imgData, imgWidth, imgScale,
```

Comment:
@myst6re can you please let me know if this might cause any side effect?

Reply:
So does this mean it's safe to merge? Sounds like a yes, but I'd like to be aligned with you.

Reply:
It is safe, I checked further and the issue is that imgWidth is already divided by imgScale above

Thanks @Sebanisu  for spotting it!

Reply:
Thanks a lot to both :) Allright, I'll wait for @Sebanisu to mark it as ready and then I'll be happy to review and merge. Cheers!

Reply:
I tested in the Balamb Hotel with tonberry style and it looks like it's working fine.